docs/src/test-api/class-testconfig.md

```diff
@@ -321,6 +321,22 @@ This path will serve as the base directory for each test file snapshot directory
 ## property: TestConfig.snapshotPathTemplate = %%-test-config-snapshot-path-template-%%
 * since: v1.28
 
+## property: TestConfig.populateGitInfo
+* since: v1.50
+- type: ?<[boolean]>
+
+Whether to populate [`property: TestConfig.metadata`] with Git info.
```

Comment:
Maybe also mention that the metadata will automatically appear in the html report, otherwise it's not clear what the use of the option?